packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -7,5 +7,6 @@ export * from './lib/common/types';
 
 export * from './lib/components';
 export * from './lib/primitives';
-export * from './lib/services';
+export * from './lib/services/authenticator-context.service';
+export * from './lib/services/state-machine.service';
```

Comment:
TIL: for production builds, services should be explicitly imported instead of index.ts else it outputs an circular dependency. Related stackoverflow -- weird behavior, but this fixed production build issues.

Reply:
I agree.  I think we should also have a clean-up task to remove all `export * from` statements and only have very explicit exports.

It's more verbose, but it's an O(n) cost.

Reply:
+1, especially at our root/index.ts file.